Makefile

```diff
@@ -49,7 +49,6 @@ verify:
 	hack/verify-generated-deep-copies.sh
 	hack/verify-generated-docs.sh
 	hack/verify-swagger-spec.sh
-	hack/verify-linkcheck.sh
```

Comment:
This link tool will not be run in Jenkins unit-test or e2e builds, it will have its own Jenkins project which runs once every day, because checking all URLs takes too much time.